docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,21 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service DCP Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to DCP rollback to zero.
+A DCP rollback is when a client connects to the Data Service with newer mutations that are not present on the Data Service.
+The client must rollback or undo some mutations to align with the mutations on the Data Service.
+If the Data Service is rolled back to 0, it means the service is attempting to resynchronize the entirety of the data set.
```

Comment:
nit: it's not the data service that gets rolled back to 0, KV is the one _sending_ the rollback - it's whatever's consuming it that gets rolled back.

Reply:
Fixed :)